R/rating.R

```diff
@@ -1,22 +1,45 @@
-fomantic_size_levels <- c("mini", "tiny", "small", "", "large", "huge", "massive")
+#' allowed sizes
+FOMANTIC_SIZE_LEVELS <- c("mini", "tiny", "small", "", "large", "huge", "massive")
 
+#' Rating
+#'
+#' Crates rating component
+#'
+#' @param name The \code{input} slot that will be used to access the value.
+#' @param label the contents of the item to display
+#' @param value initial rating value
+#' @param max maximum value
+#' @param icon An optional \code{\link{uiicon}()} to appear on the button.
```

Comment:
@dokato For rating the icon is exceptional. You need to pass exact string instead of uibutton function. The reason for that is the icon is stored as `data-icon` attribute, not as a HTML tag.

Reply:
I see... to not introduce confusion I added a way of passing both character and uiicon obj.